src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -307,6 +310,7 @@ def __init__(self, cli_ctx=None, min_profile=None, max_profile='latest',
                                                excluded_command_handler_args=EXCLUDED_PARAMS)
         self.min_profile = min_profile
         self.max_profile = max_profile
+        self.suppress_extension = None
```

Comment:
Sorry, still spinning up on Python. Why do you need to initialize this to `None` when the next line initializes it again?

Reply:
Ah, no reason. I had different logic there before. Thanks for the catch.